frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1036,6 +1045,7 @@ function ReaderRolling:_gotoPage(new_page, free_first_page, internal)
             end
         end
     end
+    self.ui:handleEvent(Event:new("PageChange", new_page))
```

Comment:
Does it really need to go through `ui:handleEvent()` ?
It feels like no other module would need to handle it, and it's ReaderRolling internal business - so here, just using self:onPageChange() would feel better/cheaper.

Reply:
On that note I'm not sure if it's particularly worthwhile having it separate at all for comprehension, but it seems the name should be `onPageChangeAnimation()`? It doesn't do anything else.

Reply:
I will figure this out once i make it work with both Cre & Mupdf